src/modules/control.cpp

```diff
@@ -34,13 +34,70 @@ static uint16_t targetVolume;
 static uint32_t totalBreathTime;
 static uint32_t targetInhalationTime;
 
+// Added
+static uint32_t targetExhalationTime;
+
+// Added
+static uint16_t targetPressure;
+
 static uint32_t measuredInhalationTime;
 static uint32_t measuredExhalationTime;
 
+// Control variables
+static uint16_t motorCommandPosition;		// Commands to low-level controller
+static uint16_t motorCommandVelocity;
```

Comment:
These do should be filled into the arguments (currently a duration and a distance) rather than via global variables.

Reply:
Ok!